cves/CVE-2017-5648.yml

```diff
@@ -10,7 +10,7 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
```

Comment:
You have six other people's CVE files in this pull request which I'm not sure is suppose to happen.

Reply:
Than you for bringing this to my attention. Sorry for the confusion.